recipes/_master_slurm_finalize.rb

```diff
@@ -22,3 +22,15 @@
   retries 3
   retry_delay 5
 end
+
+ruby_block "wait for static fleet capacity" do
+  block do
+    require 'chef/mixin/shell_out'
+
+    until shell_out!("set -o pipefail && /opt/slurm/bin/sinfo -N -h -o '%N %t' | { grep '\\-static\\-' || true; } | { grep -v -E '(idle|alloc|mix)$' || true; }").stdout.strip.empty?
```

Comment:
Minor: it would be useful to have an example of the output of the sinfo

Reply:
added!